.github/workflows/sycl-linux-run-tests.yml

```diff
@@ -371,17 +381,60 @@ jobs:
         # "test_conformance" target skips building "test_all" executable.
         ninja -C build-cts -k0 $( [ -n "$CTS_TESTS_TO_BUILD" ] && echo "$CTS_TESTS_TO_BUILD" || echo "test_conformance")
 
+    - name: Pack SYCL-CTS binaries
+      if: inputs.tests_selector == 'cts' && (success() || failure()) && inputs.cts_build_only == 'true'
```

Comment:
does `success() || failure()` do anything here? can we use `always() && !cancelled()` instead which seems similar to what we do for E2E tests